de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/allocation/impl/InventoryAllocationDestination.java

```diff
@@ -311,9 +311,15 @@ private I_M_InventoryLine getCreateInventoryLineInDispute(final I_M_InOutLine re
 
 	public List<I_M_Inventory> completeInventories()
```

Comment:
I recomment you check out the clean code book's chapter on Functions..particularly page 35ff and see if you can do something about this method name and its content